packages/react-integration/demo-app-ts/src/components/demos/FormDemo/FormDemo.tsx

```diff
@@ -165,7 +165,7 @@ export class FormDemo extends Component<FormProps, FormState> {
               ))}
             </Select>
           </FormGroup>
-          <FormSection>
+          <FormSection title="Title" titleElement="h4">
```

Comment:
how about the new isStack prop?

Reply:
Just for my own understanding, I usually tend to think of the cypress tests as testing interactivity and state changes. 
Both of these updates are just layout updates, so is that not what the snapshots and jest tests are for? All i'll be doing is adding the flags and writing 'does this classname exist on the page' sort of cypress tests. Is that what you mean?

Reply:
yup.  that is true.  the demo-app also gives us a way to test the correct typing of the api.  In this case it is just a boolean so it is not a big deal.  We have had a lot of typing issues in the past that we have only been able to to find in the demo-app.  That was more so before we converted to typescript.